xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/suggest/suggest.css

```diff
@@ -14,24 +14,32 @@ body {
 }
 
 div.suggestItems {
-  position:absolute;
+  position: absolute;
   border: 1px solid $theme.borderColor;
   border-top: none;
   background: $theme.pageContentBackgroundColor;
   z-index: 100013;
   text-align:left;
 }
+
+div.suggestItems button {
+  width: 100%;
+  background: none;
+  border: none;
+  padding: 0;
+  text-align: left;
+}
+
 div.suggestItems .hide-button-wrapper {
   line-height: 1em;
-  text-align: right;
 }
 div.suggestItems .hide-button {
   cursor: pointer;
   font-size: 0.8em;
   font-style: italic;
-  margin: 0 4px;
-  opacity: .4;
-  padding: 0 0.5em;
+  opacity: .8; /* Should eventually use a full color defined in the color theme like .text-muted */
```

Comment:
Isn't is possible to add the `text-muted` class to the `div.suggestItems .hide-button` elements?

Reply:
It's not possible here since it's a CSS file and we don't have access to inheritance like we could do for LESS.

However I didn't think about it before, but we could just add the class `.text-muted` to the span `.hide-button`.
Is it correct to use a class just to apply style like this?
@manuelleduc

Reply:
I think I can add a couple line to set the color of this element in the flamingo skin file: `action-menus.less`.

Reply:
Done :+1: